omise/omise.php

```diff
@@ -40,4 +40,100 @@ public function __construct()
     }
+
+    public function getContent()
+    {
+        $fields_form[0]['form'] = array(
+            'legend' => array(
+                'title' => $this->l('Settings')
+            ),
+            'input'  => array(
+                array(
+                    'type'     => 'switch',
+                    'label'    => $this->l('Enable/Disable'),
+                    'name'     => 'module',
+                    'is_bool'  => true,
+                    'desc'     => $this->l('Enable Omise Payment Module.'),
+                    'values'   => array(
+                        array(
+                            'id'    => 'module_enabled',
+                            'value' => 1,
+                            'label' => 'Enabled'
+                        ),
+                        array(
+                            'id'    => 'module_disabled',
+                            'value' => 0,
+                            'label' => 'Disabled'
+                        )
+                    )
+                ),
+                array(
+                    'type'     => 'switch',
+                    'label'    => $this->l('Sandbox'),
+                    'name'     => 'sandbox',
+                    'is_bool'  => true,
+                    'desc'     => $this->l('Enabling sandbox means that all your transactions will be in TEST mode.'),
+                    'values'   => array(
+                        array(
+                            'id'    => 'sandbox_on',
+                            'value' => 1,
+                            'label' => 'Enabled'
+                        ),
+                        array(
+                            'id'    => 'sandbox_off',
+                            'value' => 0,
+                            'label' => 'Disabled'
+                        )
+                    )
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Public key for test'),
+                    'name'     => 'publicKeyForTest',
+                    'required' => false,
+                    'desc'     => 'The "Test" mode public key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Secret key for test'),
+                    'name'     => 'secretKeyForTest',
+                    'required' => false,
+                    'desc'     => 'The "Test" mode secret key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
+                    'label'    => $this->l('Public key for live'),
+                    'name'     => 'publicKeyForLive',
+                    'required' => false,
+                    'desc'     => 'The "Live" mode public key can be found in Omise Dashboard.'
+                ),
+                array(
+                    'type'     => 'text',
```

Comment:
@guzzilar If this type was changed from text to password, the default PrestaShop password type is different from text.

It style was changed and its value can not be show from the setting.

![screenshot-localhost 2016-09-16 11-13-54](https://cloud.githubusercontent.com/assets/4145121/18575167/c3c18dc6-7bfe-11e6-82ae-e28df041a2b8.png)

What is the reason for this type to be password?